dev/ctests/cases/C48_ATM-gfs_atmos_prod_f000-f002.yaml

```diff
@@ -0,0 +1,102 @@
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t{{ cyc }}z.sfluxgrbf001.grib2, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t{{ cyc }}z.sfluxgrbf001.grib2]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t{{ cyc }}z.sfluxgrbf002.grib2, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/atmos/master/gfs.t{{ cyc }}z.sfluxgrbf002.grib2]
+
+output_files:
+    cmpfiles:
+        # Output atmospheric GRIB2 products at multiple resolutions for f000
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f000, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f000]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f000.idx, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f000.idx]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p50/gfs.t{{ cyc }}z.pgrb2.0p50.f000, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p50/gfs.t{{ cyc }}z.pgrb2.0p50.f000]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p50/gfs.t{{ cyc }}z.pgrb2.0p50.f000.idx, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p50/gfs.t{{ cyc }}z.pgrb2.0p50.f000.idx]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/1p00/gfs.t{{ cyc }}z.pgrb2.1p00.f000, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/1p00/gfs.t{{ cyc }}z.pgrb2.1p00.f000]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/1p00/gfs.t{{ cyc }}z.pgrb2.1p00.f000.idx, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/1p00/gfs.t{{ cyc }}z.pgrb2.1p00.f000.idx]
+
+        # Output atmospheric GRIB2 products for f001 (0p25 only - supplemental grids not at FHOUT_PGBS intervals)
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f001, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f001]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f001.idx, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f001.idx]
+
+        # Output atmospheric GRIB2 products for f002 (0p25 only - supplemental grids not at FHOUT_PGBS intervals)
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f002, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f002]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f002.idx, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/products/atmos/grib2/0p25/gfs.t{{ cyc }}z.pgrb2.0p25.f002.idx]
```

Comment:
Not sure if the test is creating this `0p25` resolution output -- do check.

Reply:
Terry.McGuinness (hfe01) 0p25 (ctest_case_updates) $ ls $PWD/gfs.t12z.pgrb2.0p25.f002
/scratch3/NCEPDEV/global/Terry.McGuinness/global-workflow_forked/dev/ctests/build/dev/ctests/RUNTESTS/COMROOT/C48_ATM-gfs_atmos_prod_f000-f002_7d0472279/gfs.20210323/12/products/atmos/grib2/0p25/gfs.t12z.pgrb2.0p25.f002
Terry.McGuinness (hfe01) 0p25 (ctest_case_updates) $